### PR TITLE
Update php.md

### PR DESCRIPTION
**Pull request by kevinhowbrook**

### CODE REVIEW

**Comment by oxyc** [comment]
You're correct that it links to the wrong URL but it should be to https://github.com/geerlingguy/ansible-role-php#readme and not to drupal-vm's readme. Could you update the PR and i'll merge it.

File: docs/configurations/php.md [hunk 1]
```diff
@@ -19,4 +19,4 @@ Remi's RPM repository is included with Drupal VM, and you can make the following
 
 If you want parity with your production environment and wish to install the default distribution packages, set `php_version: ''` inside your `config.yml` to avoid adding Remi's or Ondřej's repositories. Doing this will use the default packages set in the [`geerlingguy.php`](https://github.com/geerlingguy/ansible-role-php) Ansible role.
 
-_Note: If you're using a base OS with a PHP version older than what's assumed in the `geerlingguy.php` role, you will also need to override some of the default variables set by that role in your `config.yml`. See the [`geerlingguy.php` Ansible role's README](https://github.com/geerlingguy/drupal-vm/blob/master/provisioning/README.md) for more information._
+_Note: If you're using a base OS with a PHP version older than what's assumed in the `geerlingguy.php` role, you will also need to override some of the default variables set by that role in your `config.yml`. See the [`geerlingguy.php` Ansible role's README]( https://github.com/geerlingguy/ansible-role-php#readme) for more information._
```
Comment (oxyc) [inline docs/configurations/php.md]: You accidentally added a space before the URL

```
README]( https..
        ^
```

Comment (kevinhowbrook) [reply docs/configurations/php.md]: Not my day today! :)

**Comment by oxyc** [comment]
Thanks for cleaning up my mistakes when adding those docs! 👍

**Pull request merged**